### PR TITLE
Jenkins support

### PR DESCRIPTION
**Pull request by lslezak**
This is the Jenkins support for a simple autotools based module. The respective Jenkins build project is [yast-nfs-client-github-push](https://ci.opensuse.org/job/yast-nfs-client-github-push/).

Note: Check also the green check mark for the commit - there is a Jenkins build status result in addition to the Travis result. The link points to a build where also `rubocop` is executed (not used during normal RPM build in OBS).

Basically this is all what we need to change in YaST packages to switch from Travis to Jenkins.

(Note: so far only `master` based branches work correctly.)

### CODE REVIEW

File: package/yast2-nfs-client.spec [hunk 1]
```diff
@@ -41,6 +41,12 @@ Requires:       yast2-nfs-common >= 2.24.0
 # showmount, #150382, #286300
 Recommends:     nfs-client
 
+# Unfortunately we cannot move this to macros.yast...
```
Comment (mvidner) [inline package/yast2-nfs-client.spec]: It is not obvious why. Please reference a bug / issue that will say "bcond within macros are ignored by  prepare_build" or something like that.

Comment (mvidner) [inline package/yast2-nfs-client.spec]: This is what we found last time: https://github.com/openSUSE/obs-build/blob/5f04f59b27bc1fbd1dfa908c5287657d306fe19e/Build/Rpm.pm#L194

**Comment by mvidner** [comment]
Interesting, some tests are run twice. 
Also, "Execute rubocop"  and  "Execute check:pot" seem to be interleaved in the [output](http://ci.opensuse.org/job/yast-nfs-client-github-push/13//console).

But these are minor problems, LGTM.

**Comment by lslezak** [comment]
The problem is that for autotools packages we do not define `%check` section but the tests are run in the `%install` step (https://github.com/yast/yast-devtools/blob/master/build-tools/rpm/macros.yast#L125).

To the interleaved output: The rake progress and the warnings are printed od *stderr* while rest goes to *stdout*. The order is undefined when merging these two streams into a single output (mainly because of the buffering).

We can solve the double tests later (or switch all packages to `rake`, then this issue will disappear...).
